packages/maskbook/src/plugins/BarnBridge/BarnBridgeIcon.tsx

```diff
@@ -0,0 +1,184 @@
+import { SvgIcon, SvgIconProps } from '@material-ui/core'
+import type { FC } from 'react'
+
+export const BarnBridgeYieldFarmingIcon: FC<SvgIconProps> = (props) => (
+    <SvgIcon {...props}>
+        <svg width="50" height="50" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <circle cx="5.369" cy="16.631" r="4.369" stroke="currentColor" stroke-width="1.5" />
```

Comment:
```suggestion
            <circle cx="5.369" cy="16.631" r="4.369" stroke="currentColor" strokeWidth="1.5" />
```

Reply:
Copied this from OMGSVG as stated in an earlier comment

Reply:
but i'll change it!

Reply:
Yes, it's needed, also other properties. Otherwise, the linter will come to you. 😂

Reply:
replaced all iterations in this file